docs/example/mattermost-installation.md

```diff
@@ -221,9 +186,9 @@ To change the Mattermost installation, we need to adjust our Global policy to pr
               requires:
                 - path: "cap.type.gcp.auth.service-account"
             inject:
-              requiredTypeInstances:
+	      requiredTypeInstances:
                 - id: ${TI_ID}
-                  description: "GCP Service Account"
+		  description: "GCP Service Account"
```

Comment:
Something is broken - I think you should revert the changes in these lines:

![Screenshot 2021-09-02 at 11 32 36](https://user-images.githubusercontent.com/7155799/131820224-e4a8c7b2-edfd-42fd-8d79-ae387158d7b3.png)